page/docs/guide/advancing-time.md

```diff
@@ -20,7 +20,7 @@ Warning: block time can be set in the past and lead to unexpected behaviour!
 
 ## Increase time
 
-Increases the time offset for each generated block.
+Increases the time offset for each generated block and generate block.
```

Comment:
```suggestion
Increases the time offset for each generated block and generates a new block.
```

Reply:
fixed in bedd89c410df8d46c1a98138aa4cdbb9b53f69eb